OSRMTextInstructions/TokenType.swift

```diff
@@ -21,15 +22,17 @@ public enum TokenType: Int, CustomStringConvertible {
             type = .destination
         case "rotary_name":
             type = .rotaryName
+        case "exit":
+            type = .exitCode
         case "exit_number":
-            type = .exit
+            type = .exitIndex
```

Comment:
Same reasoning as in mapbox/MapboxDirections.swift#147, where `exitCodes` is for what’s colloquially called an exit number (`exitIdentifiers` being another candidate) and `exitIndex` remains the count of exits from the beginning of the step. I figured consistency with MapboxDirections.swift would be more important for Swift developers than consistency with the JavaScript library they’ll never use. But if you think it’s more important to keep the libraries consistent with each other (to make merging more straightforward?), I can change them to `exit` and `exitNumber` here.